### PR TITLE
docs: update docs and examples with named exports

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: examples/demos/timeslots.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 let Timeslots = ({ localizer }) => (
-  <BigCalendar
+  <Calendar
     events={events}
     step={15}
     timeslots={8}
     localizer={localizer}
-    defaultView={BigCalendar.Views.WEEK}
+    defaultView={Calendar.Views.WEEK}
```
Comment (jquense) [inline examples/demos/timeslots.js]: just `Views`

**Comment by TrySound** [comment]
Could you also publish https://github.com/jquense/date-math?

**Comment by jquense** [comment]
i published earlier this morning 👍

**Comment by TrySound** [comment]
Cool, thanks!

**Comment by jquense** [comment]
gtg here?

**Comment by TrySound** [comment]
Sure, searched though the code. Didn't find issues.

**Comment by jquense** [comment]
thanks!

**Pull request merged**